backend/microservices/videostreaming/src/middlewares/auth-middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
+import authService from '@/services/auth-service';
+
+if (!process.env.ACCESS_COOKIE_KEY) {
+  console.log('Missing ACCESS_COOKIE_KEY');
+  process.exit();
+}
+const accessTokenKey = process.env.ACCESS_COOKIE_KEY;
+
+export function authJWT(req: Request, res: Response, next: Function) {
+  const token = req.cookies[accessTokenKey];
+
+  if (token) {
+    try {
+      res.locals.userId = authService.verifyAccessToken(token).userId;
```

Comment:
can probably just use the `shared-lib` one instead, then won't need to maintain auth-service within video